packages/@webex/plugin-meetings/src/member/util.ts

```
@@ -106,21 +106,21 @@ const MemberUtil = {
    * @param {Object} participant - The locus participant object.
    * @returns {Boolean}
    */
-  isUser: (participant: any) => participant && participant.type === _USER_,
+  isUser: (participant: Participant) => participant && participant.type === _USER_,
 
   isModerator: (participant) => participant && participant.moderator,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)